pysbd/lists_item_replacer.py

```diff
@@ -110,7 +116,10 @@ def add_line_breaks_for_alphabetical_list_with_parens(self, roman_numeral=False)
         return txt
 
     def scan_lists(self, regex1, regex2, replacement, strip=False):
-        list_array = re.findall(regex1, self.text)
+        # Gerulata fix: added flag re.ASCII
+        # \d in unicode regex is not int() compatible it will return not only digits but all sort of nonsense chars (like arabic letters)
+        list_array = re.findall(regex1, self.text, re.ASCII)
```

Comment:
Ten výraz `\d` nájde všetky číslice, nie len latinské, ale aj iné, napríklad eastern arabic. Nemal by to byť problém, pretože `int()` v Pythone skonvertuje aj také číslice:

```python
int("١٢٣٤٥٦٧٨٩")
>> 123456789
```

Reply:
Je to problém. Neviem teraz nájsť presne tie znaky, ale keď som konvertoval 600GB textu tak som narazil na znaky ktor0 prejdú cez regex ale neprejdú cez CAST. Nemenil by som to keby to sústavne nepadalo na tomto riadku. (poslem ti ukazku textov ak chces)

Reply:
Tu je napríklad ukážka na ktorej celý sentencer padne pri caste na INT:
```v25. kole III.```

Reply:
Toto by som momentálne nechal v pôvodnej verzii:

```suggestion
        list_array = re.findall(regex1, self.text)
```